services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/oauth/OIDCTokenHelperImplTest.java

```diff
@@ -148,14 +150,15 @@ public void testGenerateOIDCIdentityToken() throws Exception {
 		assertTrue(claims.get(OIDCClaimName.email_verified.name(), Boolean.class));
 		assertEquals("University of Example", claims.get(OIDCClaimName.company.name(), String.class));
 		assertEquals(TOKEN_ID, claims.getId());
+		assertEquals(TokenType.OIDC_ID.name(), claims.get(OIDCClaimName.token_type.name(), String.class));
 		
 		// This checks the other fields set in the method under test
-		clientValidation(oidcToken, NONCE);
+		jwtValidation(oidcToken, false, NONCE);
 	}
 
     // let's check that our JWTs fulfill the OIDC spec by checking that they meet the requirements for client validation:	
 	// https://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation
-	private void clientValidation(String jwtString, String nonce) throws ParseException {
+	private void jwtValidation(String jwtString, boolean isPersonalAccessToken, String nonce) throws ParseException {
```

Comment:
add a comment that this checks the OIDC spec when isPersonalAccessToken==false.  When ==true, the checks are similar but not identical.